### DISCUSSION TITLE
[Support] Instance Stuck in Starting

### DISCUSSION THREAD

**Discussion by Hyperclaw79** [pockethost/pockethost#204] [Q&A]
I had a fresh instance for [https://ht-dev.pockethost.io](https://ht-dev.pockethost.io) (instance ID 80r6po90ms7kiar) which was working perfectly fine.
I have used FileZilla Windows Client to upload my local pb_data via FTP.
After that, the instance has been stuck on the Starting stage.
<img width="165" alt="image" src="https://user-images.githubusercontent.com/29298411/235308825-08a9c248-3cfa-42de-a392-10fc30c805ac.png">
I can't see any errors in the logs either. 
<img width="1006" alt="image" src="https://user-images.githubusercontent.com/29298411/235308867-00c4cc1e-94e0-4077-803b-900f060c0f0d.png">

Is there any way to get the verbose logs or force a restart of the instance?

**Comment by benallfree** [comment]
@Hyperclaw79 I've reset your `starting` status to `idle`, this should allow the system to launch it again. I'm curious though, if uploading to `pt_data` did not work correctly. Has the instance worked at all since you uploaded new data via FTP?

**Reply by Hyperclaw79** [reply]
@benallfree Thanks for resetting it. It's been stuck like this right after the first data upload and this is on a fresh instance.
What's surprising though is that i have created another instance and followed the same exact steps and that one is working fine.
Would you be able to find out from server side logs what could've happened for the first instance?

**Reply by benallfree** [reply]
I'm not sure the logging is that detailed yet, it would be a lot to sift through. The easier approach is to monitor the logs while attempting to reproduce the problem. But if you say it's working fine now, that's probably good enough for me in this case :)

**Reply by Hyperclaw79** [reply]
@benallfree Unfortunately, i checked the instance again and it's again stuck in the starting stage...
Would it be possible to wipe out this instance so that i can recreate it?
The other instance is still working fine though without issues. The only drawback is the instance name which i can live with if it's too much effort to wipe out the older one. (The older one has the name i prefer 😅)

**Reply by benallfree** [reply]
Before I delete it, I'd like to see if I can carve out some time to see what error message it's producing. It seems likely related to a corrupted db or something like that. Perhaps I can expose a few helpful error messages to the UI so you can see why something like that would be failing.

**Reply by Hyperclaw79** [reply]
Sounds perfect!
Good luck finding the cause

**Reply by jaxsonsprinkles** [reply]
@benallfree could you please reset/delete mine as well? I am having the same issue as OP. The id is mlbgab2eifrorxf.
